packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1497,36 +1497,38 @@ export default class Meeting extends StatelessWebexPlugin {
       // const {transceiver} = event;
       const streams = event.streams[0];
```

Comment:
Can we rename this to `eventStream` instead? I was confused on line 1501 that did:
`this.mediaProperties.remoteStream = streams;`